app/views/layouts/_site_nav.html.erb

```diff
@@ -1,14 +1,21 @@
 <div class="wrap-outer-header-local layout-band">
   <div class="wrap-header-local">
-    <div class="local-identity">
-      <h2 class="title title-site"><a href="/">TIMDEX UI</a></h2>
-    </div>
-    <div class="wrap-local-nav">
-      <div class="wrap-bar">
-        <nav class="local-nav" aria-label="Main menu">
-          <%= nav_link_to("Home", root_path) %>
-        </nav>
+    <% unless Flipflop.enabled?(:gdt) %>
```

Comment:
Not blocking, but I wonder if we should have all the UIs just move to the new name in the header? Or alternatively not tie this to GDT?

Reply:
That's certainly my preference, but I didn't want to make that any assumptions about how we want the other TIMDEX UI apps to appear. If you're good with it, I'll just remove this bit.